new/LIT7413QeneAyte.xml

```diff
@@ -0,0 +1,78 @@
+                <title xml:lang="gez" type="normalized" corresp="#t1">ʾAyte ʾaḥawwǝr wa-ʾayte ʾǝbawwǝʾ ʾƎgzeʾtǝya la-ṣabāʿot waldǝki soba 
+                    tafaṣṣama bāhlu (Qǝne of the śǝllāse-type)</title>
+                <title xml:lang="en" corresp="#t1">Where shall I go and where shall I enter, My Lady, when the speech about Zebaoth, your Son, 
+                    is finished?…</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>The author of the qǝne is not indicated in the manuscript. The type of the qǝne is indicated as 
+                    <foreign xml:lang="gez">ሥላሴ፡</foreign>.</p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Poetry"/>
+                    <term key="Qene"/>
+                    <term key="Sellase"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2025-04-29">Created entity</change>
+            <change when="2025-04-29" who="CH">Corrected title and edition according to Jonas Karlssons requests</change>
+            <change when="2025-04-30" who="CH">Updated abstract for more conformity to other Bm records</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="edition" xml:lang="gez">
+                <note>This edition is based on <ref type="mss" corresp="PetermannIINachtrag24"/>, fol. 7rb. Division into verses is indicated 
+                    with ። .</note>
+                <ab>
+                    ሥላሴ፡
+                    <l n="1"><choice resp="CH"><sic>ዓ</sic><corr>አ</corr></choice>ይቴ፡ አሐውር፡ 
```

Comment:
Again, I would not normalize. These deviations are normal and do not really prevent from understanding, I do not htink we correct them. Also, sometimes it has a meaning, like the play of words etc.